recipes/pygmo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1.7" %}
+
+package:
+  name: pygmo
+  version: {{ version }}
+
+source:
+  fn: pagmo-{{ version }}.tar.gz
+  url: https://github.com/esa/pagmo/archive/{{ version }}.tar.gz
+  sha256: 6d8fab89ef9d5d5f30f148225bf9b84b2e5a38997f3d68b85547840e9fd95172
+
+build:
+  number: 0
+
+  # https://github.com/esa/pagmo/issues/94
+  skip: true  # [win or osx and py34]
```

Comment:
Little confused by this skip. We are trying to skip Windows, which is clear.

However, what is this last part? Are we trying to skip Python 3.4 on OS X? Could you please explain why?